modules/flowable-task-service/src/main/java/org/flowable/task/service/impl/TaskQueryImpl.java

```diff
@@ -200,6 +202,34 @@ public TaskQueryImpl taskName(String name) {
         return this;
     }
 
+    /**
+     * Only select tasks with a task id that is in the given list
+     *
+     * @param taskIdList
+     * @throws FlowableIllegalArgumentException When task id list is empty or <code>null</code> or contains <code>null String</code>.
+     */
+    @Override
+    public TaskQuery taskIdIn(Collection<String> taskIdList) {
+        if (taskIdList == null) {
+            throw new FlowableIllegalArgumentException("Task id list is null");
+        }
+        if (taskIdList.isEmpty()) {
+            throw new FlowableIllegalArgumentException("Task id list is empty");
+        }
+        for (String taskId : taskIdList) {
+            if (taskId == null) {
+                throw new FlowableIllegalArgumentException("None of the given task ids can be null");
+            }
+        }
```

Comment:
Same comment as in the history query. We don't do this for any other collection, so we can remove it